gio/Gir.toml

```diff
@@ -833,7 +833,12 @@ generate_builder = true
     manual = true
     [[object.function]]
     name = "find_with_equal_func"
-    ignore = true # See https://gitlab.gnome.org/GNOME/glib/-/issues/2447
+    # calls find_with_equal_func_full
```

Comment:
I would mark this one as ignored as mark the other one as renamed instead

Reply:
I did it this way because the docs are more useful then

Reply:
The FFI calls aren't renamed?

Reply:
How do you mean?

Reply:
`find_with_equal_func` calls `g_list_store_find_with_equal_func_full` because the non-`_full` variant is not usable for bindings and the difference between the two functions is irrelevant for the bindings user (i.e. the extra sentence in the docs of the `_full` variant are just confusing for the bindings user).

Reply:
Oh I see. I thought the code generated by gir was invalid because it was badly renamed or something like that.